<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PhoneBookWidget</class>
 <widget class="QWidget" name="PhoneBookWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>758</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>450</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>641</width>
     <height>411</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="4,2">
    <item>
     <widget class="QTableView" name="tableView"/>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string/>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>171</width>
         <height>361</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,2">
        <property name="topMargin">
         <number>20</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>10</number>
          </property>
          <item>
           <widget class="QLineEdit" name="lineEdit">
            <property name="placeholderText">
             <string>请输入姓名或者号码进行查询</string>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignHCenter">
           <widget class="QPushButton" name="pushButtonSearch">
            <property name="text">
             <string>查询</string>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
           <widget class="QPushButton" name="pushButtonShowAll">
            <property name="text">
             <string>显示全部</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBoxAdmin">
          <property name="title">
           <string/>
          </property>
          <widget class="QWidget" name="layoutWidget">
           <property name="geometry">
            <rect>
             <x>50</x>
             <y>20</y>
             <width>77</width>
             <height>139</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="spacing">
             <number>15</number>
            </property>
            <item>
             <widget class="QPushButton" name="pushButtonAdd">
              <property name="text">
               <string>增加</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonDelete">
              <property name="text">
               <string>删除</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonMod">
              <property name="text">
               <string>修改</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>pushButtonSearch</tabstop>
  <tabstop>pushButtonShowAll</tabstop>
  <tabstop>pushButtonAdd</tabstop>
  <tabstop>pushButtonDelete</tabstop>
  <tabstop>pushButtonMod</tabstop>
  <tabstop>tableView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
